lib/providers/services/http_service.dart

```diff
@@ -51,6 +52,32 @@ class HttpService {
     }
   }
 
+  Future<List<Permission>> getAccountPermissions() async {
```

Comment:
Honestly, if we simply put this get method into eos_service, then we can also provide all our utlitity methods there which are currently in the widgets (add and remove guard.seeds permission)

I think this should just be in eos_service

What do you think @7flash